frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -41,35 +62,32 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "workerman-postgres",
+      "display_name": "workerman [jit, pgsql]",
       "notes": "",
       "versus": "php"
     },
-    "async": {
-      "db_url": "/db",
-      "fortune_url": "/fortunes",
+    "without-jit": {
```

Comment:
Please don't change the name.
Use the same names, or all the history will be lost.

Reply:
Now I'm busy.
Give sometime and I'll check it.

Reply:
Ok thanks.